src/service/repo/signedMessageRepo.ts

```diff
@@ -89,30 +89,39 @@ async function sendSignedMessage(
   }
 }
 
-async function getAllSignedChannels() {
+async function getAllSignedChannels(timeStamp: string | undefined) {
+  const url = timeStamp
+    ? `${baseUrl.getAllSignedChannels}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllSignedChannels;
   try {
-    const response = await api.get(baseUrl.getAllSignedChannels);
-
+    const response = await api.get(url);
     if (response.status === 200) {
+      console.log('SUCCESS URL', url);
       return Promise.resolve(response.data?.data);
     }
 
     return Promise.reject(response.data?.status);
   } catch (e) {
+    console.log('FAILED URL', url);
     console.log(e);
     return Promise.reject(e);
   }
 }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `getAllSignedChannels` function, you are using console.log statements for debugging purposes. While this is fine during development, it's not a good practice to leave them in production code as they can expose sensitive information. Consider removing these or replacing them with a proper logging library that can be configured to mute logs in production.

```diff
- 99:       console.log('SUCCESS URL', url);
...
- 105:     console.log('FAILED URL', url);
- 106:      console.log(e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->